.gitignore

```diff
@@ -6,3 +6,4 @@ tmp/
 .coverage
 /htmlcov
 /.dir-locals.el
+.tox/
```

Comment:
Remove trailing slash, as not consistent with previous directory entries, e.g. htmlcov
Could also add a leading slash, to pin it to the top level directory, but I'm not really bothered.